src/features/ITGovernance/Admin/GRBReview/GRBReviewStatusCard/index.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Card, CardBody, CardHeader } from '@trussworks/react-uswds';
+import classNames from 'classnames';
+import {
+  SystemIntakeFragmentFragment,
+  SystemIntakeGRBReviewType
+} from 'gql/generated/graphql';
+import ITGovAdminContext from 'wrappers/ITGovAdminContext/ITGovAdminContext';
+
+import UswdsReactLink from 'components/LinkWrapper';
+import Tag from 'components/Tag';
+import { formatDateUtc } from 'utils/date';
+
+// TODO: Temp status type;
+export enum GRBReviewStatus {
+  SCHEDULED,
+  IN_PROGRESS,
+  COMPLETED
+}
+
+export type GRBReviewStatusCardProps = {
+  grbReviewType: SystemIntakeFragmentFragment['grbReviewType'];
+  grbDate?: SystemIntakeFragmentFragment['grbDate'];
+  grbReviewStatus: GRBReviewStatus;
+  className?: string;
+};
+
+const renderBGColor = (grbReviewStatus: GRBReviewStatus) => {
+  if (grbReviewStatus === GRBReviewStatus.COMPLETED) {
+    return 'bg-success-lighter';
+  }
+  return 'bg-primary-lighter';
+};
+
+const GRBReviewStatusCard = ({
+  grbReviewType,
+  grbDate,
+  grbReviewStatus,
+  className
+}: GRBReviewStatusCardProps) => {
+  const { t } = useTranslation('grbReview');
+
+  const isITGovAdmin = useContext(ITGovAdminContext);
+
+  const StandardCard = (
+    <Card
+      className={classNames(className)}
+      containerProps={{
+        className: classNames(
+          'margin-0 border-none radius-md',
+          renderBGColor(grbReviewStatus)
+        )
+      }}
+    >
+      <CardHeader>
+        <h3 className="display-inline-block margin-right-2 margin-bottom-0">
+          {t('statusCard.standardHeading')}
+        </h3>
+      </CardHeader>
+
+      <CardBody>
+        {/* Status Section */}
+        <span
+          className={classNames('display-flex', {
+            'border-bottom-1px border-primary-light margin-bottom-2 padding-bottom-2':
+              grbReviewStatus !== GRBReviewStatus.COMPLETED
+          })}
+        >
+          <h4 className="margin-0 margin-right-1 margin-top-2px">
+            {t('statusCard.reviewStatus')}
+          </h4>
+
+          <Tag className="bg-white">
+            {t(`statusCard.grbReviewStatus.${grbReviewStatus}`)}
+          </Tag>
+        </span>
+
+        {/* Meeting Details */}
+        {grbReviewStatus !== GRBReviewStatus.COMPLETED && (
+          <span>
+            <h4 className="margin-0 margin-right-1 margin-top-2px margin-bottom-05">
+              {t('statusCard.grbMeeting')}
+            </h4>
+
+            <div className="easi-body-large">
+              {formatDateUtc(grbDate, 'MM/dd/yyyy')}
+            </div>
+
+            {isITGovAdmin && (
+              <UswdsReactLink
+                to="./dates"
+                className="usa-button usa-button--outline margin-top-2"
```

Comment:
Spacing looks a little off on this button - I think `margin-top-1` matches the Figma.